docker/entrypoint.sh

```diff
@@ -27,11 +27,18 @@ function init_if_possible () {
 
 function generate_password () {
     set +o pipefail
-    < /dev/urandom tr -dc A-Za-z0-9 | head -c40
+    < /dev/urandom tr -dc A-Za-z0-9 | head -c256
```

Comment:
Let's not increase the password length here. 40 characters is long enough.

Reply:
force pushed fixing everything